.gitlab-ci/debian-install.sh

```diff
@@ -61,7 +61,6 @@ apt-get install -y \
 	libudev-dev \
 	libunwind-dev \
 	libwayland-dev \
-	libx11-dev \
```

Comment:
Why are you removing libx11-dev, but not libx11-xcb-dev ?

Reply:
Removed, still builds. Thanks!